### PR TITLE
chore: added keycloak authentication and aws parameters TISNEW-4361

### PR DESCRIPTION
**Pull request by jayanta2018**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,7 +40,7 @@ gradle-app.setting
 .gradletasknamecache
 
 # # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
+#gradle/wrapper/gradle-wrapper.properties
```
Comment (Judge40) [inline .gitignore:43]: Small formatting issue.

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/config/KeyCloakConfig.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.nhs.hee.tis.revalidation.integration.config;
+
+import org.springframework.context.annotation.Bean;
```
Comment (Judge40) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/config/KeyCloakConfig.java:24]: I suspect import order may be flagged by sonar for these.

File: .gitignore (hunk 1 above)
Comment (jayanta2018) [reply .gitignore]: Done

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/config/KeyCloakConfig.java (hunk 2 above)
Comment (jayanta2018) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/config/KeyCloakConfig.java]: Import order is changed

**Pull request merged**